doc/sphinx/source/developer_guide/logging.rst

```diff
@@ -0,0 +1,264 @@
+
+  <conf>
+    <aircraft
+     name="Bebop_default"
+     ac_id="20"
+     airframe="airframes/examples/bebop.xml"
+     radio="radios/dummy.xml"
+     telemetry="telemetry/default_rotorcraft.xml"
+     flight_plan="flight_plans/rotorcraft_basic.xml"
+     settings="settings/rotorcraft_basic.xml settings/control/rotorcraft_speed.xml"
+     settings_modules="modules/ahrs_float_mlkf.xml modules/air_data.xml modules/bebop_ae_awb.xml modules/bebop_cam.xml modules/geo_mag.xml modules/gps.xml modules/guidance_rotorcraft.xml modules/imu_common.xml modules/ins_extended.xml modules/nav_basic_rotorcraft.xml modules/stabilization_int_quat.xml modules/video_rtp_stream.xml"
+     gui_color="#ffffbc3bce5b"
+    />
+  </conf>
+
+Decoding FlightRecorder logs
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+To download and convert the data, you need to first connect the SD card to your laptop, navigate to the
+FLIGHT_RECORDER folder and transfer to your computer the relevant logs, named ``fr_XXXX.LOG``. To convert
+this binary file into Pprzlog format files the program ``sw/logalizer/sd2log`` has to be used.
+Make sure the environment variables are set before running Paparazzi executables from the commandline.
+They can be set in your Terminal by using
+
+.. code-block:: bash
+
+  export PAPARAZZI_HOME=~/paparazzi
+  export PAPARAZZI_SRC=~/paparazzi
+
+.. tip::
+
+  You can add the ``export`` lines above to your ``.bashrc`` file to automatically set the environment variables
+  every time a new Terminal is opened.
+
+Once the environment variables are set you can run ``sd2log`` in your terminal:
+
+.. code-block:: bash
+
+  ~/paparazzi/sw/logalizer/sd2log fr_XXXX.LOG
+
+This will produce the Paparazzi .log, .data, and .tlm files that are stored in ``var/logs``. It creates a timestamp from the .tlm and changes the filename
+to the take-off time if a GPS mesage with correct time was available in the file, or the current local PC time if no
+GPS was available. The .log file will be recreated either from the current configuration or the MD5-labeled files that
+are stored in ``var/conf`` each time you build an aircraft.
+
+.. note::
+
+  For the decoding process to work properly you must run ``sd2log`` from the same folder that contains the
+  ``messages.xml`` file used during compilation of the aircraft. This is often not an issue, but you may run into
+  problems if you work with multiple branches within your paparazzi repository.
+
+Pprzlog format
+-----------------
+
+The Pprzlog format creates multiple files that can be used to analyse flight data and replay the flight. A log is
+split into the following files:
+
+- A ``.log`` file, an XML file, which contains a copy of the whole configuration (airframes, flight plans, ...)
+- A ``.data`` file, an ascii file, which contains the list of the received messages. Each message is time-stamped in
+  milliseconds since the creation of the file and marked with the ID of the sending aircraft
+- A ``.tlm`` file, ???
```

Comment:
the `tlm` file is a copy of the original `.LOG` file, with the same naming `data_time.*`. It makes it easier to associate with decoded log files.
By the way, a recommended use is to directly call `sd2log` on the file still on the SD card mounted as a mass storage, it is not really needed to copy it first on the computer.